### PR TITLE
Signpost CronJob docs from Job

### LINKED ISSUE #12601: CronJob reference mentions support from Kubernetes 1.4
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#cron-jobs mentions that `Job` is available in Kubernetes 1.4 and later.
`CronJob` is now a stable feature, the docs don't need to mention that it once wasn't.

**Proposed Solution:**
Update this section of the page.

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/

v1.13

### PR DESCRIPTION
**Pull request by sftim**
CronJob is now stable. Signpost users to the relevant documentation.

Fixes #12601

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md [hunk 1]
```diff
@@ -448,8 +448,8 @@ driver, and then cleans up.
 An advantage of this approach is that the overall process gets the completion guarantee of a Job
 object, but complete control over what pods are created and how work is assigned to them.
 
-## Cron Jobs
+## Scheduled jobs {#cron-jobs}
```
Comment (soltysh) [inline content/en/docs/concepts/workloads/controllers/jobs-run-to-completion.md]: Cron Jobs

**Comment by zhangxiaoyu-zidif** [comment]
see https://docs.openshift.com/container-platform/3.11/dev_guide/cron_jobs.html#dev-guide-cron-jobs `jobs` should be caps on too.

**Comment by zparnold** [comment]
@sftim once you make the relevant change @zhangxiaoyu-zidif suggested, this is ready to merge

**Comment by sftim** [comment]
@zhangxiaoyu-zidif - I'm not clear on how to identify which uses of “jobs” you recommend capitalizing. Perhaps you're suggesting capitalizing every use? I'm not sure.

**Comment by sftim** [comment]
Actually it does make sense to capitalize Job everywhere this PR touches. Done.

**Pull request merged**